### PR TITLE
Instructs for github actions

### PR DESCRIPTION
**Pull request by realPy**
Add readme instruct  for github actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,45 @@ install:
...
+PS: adjust the go version you need in go-version section
+
+```
+on: [push, pull_request]
+name: Unit Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.17.x]
```
Comment (agnivade) [inline README.md:91]: It might be better to just mention `1.xx.x`

File: README.md [hunk 2]
```diff
@@ -73,6 +73,45 @@ install:
...
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Install chrome
+      uses: browser-actions/setup-chrome@latest
+    - name: Run chrome headless
+      run:  google-chrome --no-sandbox --disable-gpu --headless --remote-debugging-address=0.0.0.0 --remote-debugging-port=9222 &
+    - name: Install dep
+      run: go install github.com/agnivade/wasmbrowsertest@latest
```
Comment (agnivade) [inline README.md:104]: Can we also change line 69 to this?

File: README.md [hunk 3]
```diff
@@ -73,6 +73,45 @@ install:
...
+    - name: Setup wasmexec
+      run: mv $(go env GOPATH)/bin/wasmbrowsertest $(go env GOPATH)/bin/go_js_wasm_exec
+    - name: Keep only env useful
+      run: echo "GOPATH=$(go env GOPATH) GOCACHE=$(go env GOCACHE) GOROOT=$(go env GOROOT) PATH=$(echo $PATH)" >/opt/.env    
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Test
+      run:  env -i $(cat /opt/.env) GOOS=js GOARCH=wasm go test ./...
```
Comment (agnivade) [inline README.md:112]: Let's remove these lines. The environment variables hack is only necessary until the dot release is out. And the Travis section doesn't mention the "go test" step as well. Let's just keep both section only till the setup part.

File: README.md [hunk 4]
```diff
@@ -73,6 +73,45 @@ install:
...
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Install chrome
+      uses: browser-actions/setup-chrome@latest
+    - name: Run chrome headless
+      run:  google-chrome --no-sandbox --disable-gpu --headless --remote-debugging-address=0.0.0.0 --remote-debugging-port=9222 &
```
Comment (agnivade) [inline README.md:102]: I don't believe you need to run Google Chrome. The tool will automatically spawn an instance.

**Review by agnivade — COMMENTED** [review]
Thank you @realPy. Just some small suggestions.

File: README.md (hunk 2 above)
Comment (realPy) [reply README.md:104]: I can , but go get is deprecated for install binary

Comment (agnivade) [reply README.md]: Yes I know, and that's what I am suggesting to do. :wink: To change `go get` to `go install` in line 69. Anyways, I can do that.

**Review by agnivade — APPROVED** [review]
Thank you!

**Pull request merged**